src/app/log-in/log-in.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="container-fluid">
+    <div class="row">
+        <!-- Log in page -->
+        <div class="col-md-6">
+            <div class=" col-md-12 d-flex logInForm">
+                <div class="col-md-8">
+                    <div class="mt-5">
+                        <h2 class="text-center">LOGIN</h2>
+                    </div>
+                    <div class="text-center"
+                    style="color: #636364;">
+                        <small>Please enter your credintials</small>
+                    </div>
+                    <!-- <div *ngIf=""></div> -->
+                    <div *ngIf="isError" class="alert alert-danger mt-2 rounded-alert text-center fw-bold ">
+                        <small>Please enter a correct national ID/password</small>
+                    </div>
+                    <!-- form -->
+                    <form [formGroup]="logInForm" (ngSubmit)="onSubmit()">
+                        <div class="mt-3">
+                            <label class="ms-1 label-text">Username</label>
+                            <input type="number" class="form-control mt-1 rounded-input"
+                                placeholder="Enter your username" formControlName="username"
+                                [ngClass]="{'rounded-input-error': (isError)}">
+                        </div>
+                        <div class="mt-3">
+                            <label class="ms-1 label-text">Password</label>
+                            <input type="password" class="form-control mt-1 rounded-input" placeholder="********"
+                                formControlName="password" [ngClass]="{'rounded-input-error': (isError)}">
+                        </div>
+                        <div class="mt-2 row gx-5">
+                            <div class="col-sm-6 col-md-8">
+                                <input type="checkbox" formControlName="rememberMe">
+                                <label class="ms-1 label-text">Remember me</label>
+                            </div>
+                            <div class="text-end col-6 col-md-4 label-text">
+                                Forgot password
+                            </div>
+                        </div>
+                        <div>
+                            <button class="btn mt-3 col-md-12 text-light rounded-btn " (click)="onSubmit()">Sign
+                                In</button>
+                        </div>
+                    </form>
+                    <!-- form end -->
+                </div>
+            </div>
+        </div>
+        <!-- side page -->
+        <div class="col-md-6 side-bacground ">
+            <div>
+                <div class="container mt-5 text-center">
+                    <img src="assets/images/logoimg.png" alt="Example Image" class="img-fluid"
+                        style="width: auto; height: 100%;">
+                </div>
+
+            </div>
+
+
+        </div>
```

Comment:
```suggestion
`<div class="col-md-6 side-bacground d-flex align-items-center justify-content-center">`
     `<img src="assets/images/logoimg.png" alt="Example Image" class="img-fluid">`
 `</div>`
```

Reply:
a simpler code that preserves the logo's quality.
[original code]
![image](https://github.com/MohammadWS/Tahakom-skill-assessment/assets/174691803/494302a2-ed5f-422f-981e-9b0a8b77da77)
[suggested code]
![image](https://github.com/MohammadWS/Tahakom-skill-assessment/assets/174691803/c784f2d0-574f-4877-a765-fa881345427e)